### PR TITLE
New plugin Gitty for Github

### PR DESCRIPTION
**Pull request by collegboi**
I created a new plugin called Gitty, which finds your local repos and shows the status.

### CODE REVIEW

**Comment by tresni** [comment]
We generally require that a plugin be contained within a single file.  The plugin repo (and generally Bitbar) are not designed to handle multi-file scripts or scripts with dependancies/resources.  I'll admit this is a pretty cool way to make that work anyway, but I'm not comfortable merging this in as is.  I've tagged @matryer to take a look.

**Comment by collegboi** [comment]
Let me know if you wish for it all to be in a single file. Thanks for your feedback.

**Comment by matryer** [comment]
@collegboi we do. It has lots of benefits, including simplicity.

**Comment by collegboi** [comment]
@tresni The build failed due to PIP not found. Can you re-run the tests?

**Comment by tresni** [comment]
Looks like there may have been a change to how pip shows up on Travis. I'll investigate and rerun once I think I have it resolved.

**Comment by tresni** [comment]
This is due to https://github.com/Homebrew/homebrew-core/pull/14408 which changed python executables to now be named x2 (e.g. python2, pip2, etc)  I'll fix the test scripts and see what we can do.  You may need to rebase your changes once I get the test script fixed.

File: Dev/Gitty/git_python.5m.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# <bitbar.title>Gitty</bitbar.title>
+# <bitbar.version>v0.1</bitbar.version>
+# <bitbar.author>Timothy Barnard</bitbar.author>
+# <bitbar.author.github>collegboi</bitbar.author.github>
+# <bitbar.image>https://raw.githubusercontent.com/collegboi/Bitbar-Gitty/master/image1.png</bitbar.image>
```
Comment (tresni) [inline Dev/Gitty/git_python.5m.py]: This image doesn't exist anymore.  Should either replace it with one that does (recommended) or remove the metadata.

**Comment by tresni** [comment]
Checks fail due to the image metadata.  I've commented on.  Otherwise you can either rebase on the current master or just push the necessary change and I'll test locally.

**Pull request merged**